requirements.txt

```diff
@@ -18,7 +18,7 @@ trio==0.19.0
 
 # Documentation
 mkdocs==1.3.0
-mkdocs-material==8.2.8
+mkdocs-material==8.3.3
```

Comment:
This change is not related to this PR :), In my opinion, don't put irrelevant changes together—just advice.